docs/workspaces.md

```diff
@@ -381,6 +381,12 @@ significantly. We do not recommend using them in clusters larger than several hu
 node in the cluster must have an appropriate label matching `topologyKey`. If some or all nodes
 are missing the specified `topologyKey` label, it can lead to unintended behavior.
 
+**Note:** Any time during the execution of a `pipelineRun`, if the node where a placeholder Affinity Assistant pod and
```

Comment:
This might be a better fit for the developer docs (https://github.com/tektoncd/pipeline/tree/main/docs/developers)

Reply:
Sure, we can add a little bit more details in developers doc but I would like to keep this small **Note** here too. This is directly related to the impact on a running pipeline with affinity assistant enabled.

Reply:
Added a developer doc for the same